site/docs/components/date-picker/examples.mdx

```diff
@@ -36,6 +36,14 @@ Date picker can show validation states (warning, success, error) with the prop `
 The Date Range Picker enables selection of start and end dates, using the input fields or two calendars displayed.
 
 </LivePreview>
+
+<LivePreview componentName="date-picker" exampleName="CompactRange">
+## Compact Date range
+
+The Compact Date Range Picker enables selection of start and end dates, using the input fields or one calendars displayed in case of mobile viewport.
```

Comment:
I would imagine the limited space might not only be the result of it being in mobile but potentially in a small window or even a small section of the application

maybe something like: 
```suggestion
The Compact Date Range Picker renders a single calendar for a range selection. We recommend using it when the space is limited.
```

Reply:
done